pkgs/by-name/cl/clprover/package.nix

```
@@ -0,0 +1,50 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "CLProver++";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "PaulGainer";
+    repo = "CLProverPlusPlus";
+    rev = "66bf6dc19d77094cc06eac3f30a3630ba830ac86";
+    sha256 = "sha256-UZ5e11wGuKyVrG+7hZJY6OmN6Y1mg43xPuvXPRVNKNw=";
+  };
+
+  configurePhase = lib.optional stdenv.hostPlatform.isCygwin ''
+    runHook preConfigure
+
+    sed -i 's/\(TARGET_OS *:= *\)LINUX/\1WINDOWS/g' Makefile
+
+    runHook postConfigure
+  '';
+
+  preBuild = ''
+    mkdir bin
+  '';
+
```

```suggestion

  makeFlags = [
    "CC=${lib.getBin stdenv.cc}/bin/${stdenv.cc.targetPrefix}cc"
  ];
```
This partially fixes cross-compiling.